pkg/sidecarinjector/mutate.go

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sidecarinjector
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/golang/glog"
+	"k8s.io/api/admission/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	admissionWebhookAnnotationInjectKey = "sidecar-injector-webhook.nsm.ligato.io/inject"
```

Comment:
Note that in other places, we've used `networkservicemesh.io` instead of `nsm.ligato.io`, for consistency, I'd make that change here too.